src/frame/js/components/Action.jsx

```diff
@@ -139,8 +140,8 @@ export class ActionComponent extends Component {
                                        email={ user.email }
                                        amount={ amount }
                                        currency={ currency.toUpperCase() }
-                                       name={ stripeAccount.appName }
-                                       image={ stripeAccount.iconUrl }
```

Comment:
Is there a service or action that was fetching this data that should be deleted also?